cmd/proxy/actions/app.go

```diff
@@ -75,12 +75,18 @@ func App(conf *config.Config) (*buffalo.App, error) {
 		app = app.Group(prefix)
 	}
 
-	// Register exporter to export traces
-	exporter, err := observ.RegisterTraceExporter(conf.TraceExporterURL, Service, ENV)
+	// RegisterExporter is used for registering and exporting traces from opencensus
```

Comment:
will register an exporter where we will export our traces to.
Remove opencensus here.